javascripts/main.js

```diff
@@ -47,19 +73,26 @@
       return;
     }
 
-    count = $('<span class="count"><img src="images/octocat-spinner-32.gif" /></span>').appendTo(a);
-    $.ajax(url)
-      .done(function(data, textStatus, jqXHR) {
-        count.html(data && typeof data.length === 'number' ? data.length.toString() : '?');
-      })
-      .fail(function(jqXHR, textStatus, errorThrown) {
-        var rateLimited = jqXHR.getResponseHeader('X-RateLimit-Remaining') === '0'
-          , rateLimitReset = rateLimited && new Date(1000 * +jqXHR.getResponseHeader('X-RateLimit-Reset'))
-          , message = rateLimitReset ? 'GitHub rate limit met. Reset at ' + rateLimitReset.toLocaleTimeString() + '.' :
-                      'Could not get issue count from GitHub: ' + ((jqXHR.responseJSON && jqXHR.responseJSON.message) || errorThrown) + '.';
-        count.html('?');
-        count.attr('title', message);
-      });
+    count = $('<span class="count"><img src="images/octocat-spinner-32.gif" /></span>').appendTo(a);
+    var cached = storage.get(gh[1]);
+    if (cached && cached.date >= (new Date() - 1000 * 60 * 60 * 24)) {
+        count.html(cached.count)
+    } else {
```

Comment:
Instead of `if`/`else` can we just `return` within the `if` to avoid the extra level of indentation?